Source/ACE/Entity/Player.cs

```diff
@@ -1605,5 +1605,30 @@ public void ClearDelayedTeleport()
             DelayedTeleportTime = DateTime.MinValue;
             WaitingForDelayedTeleport = false;
         }
+
+        override public void GameLoopUpdate()
+        {
+            uint oldHealth = Health.Current;
```

Comment:
I'm ok with this in general, but I had envisioned making an "IsDirty" flag with the CreatureAbility object, then sending an update when it's dirty.